packages/iov-ethereum/webpack.web.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = [
       filename: "tests.js",
     },
     plugins: [
-      new webpack.EnvironmentPlugin(['LONG_RUNNING_ENABLED']),
+      new webpack.EnvironmentPlugin(['ETH_ENV']),
```

Comment:
We need ETHEREUM_ENABLED as well. Otherwise the browser tests do not see the environment variable